src/store/epics/wallets.ts

```diff
@@ -54,18 +58,25 @@ export const unlockWallet$: RootEpic = (action$, state$, { decryptPrivateKey })
     filter(isActionOf(unlockWalletRequest)),
     withLatestFrom(state$),
     map(([_, state]) => {
+      const now = Date.now()
+      if (failureAttemptHandler(state.wallets.failureAttemptTimestamps, now).afterLockInTime) {
+        return clearWalletFailures()
+      } else if (failureAttemptHandler(state.wallets.failureAttemptTimestamps, now).withinLockInTime) {
+        return tooManyFailuresMessage()
```

Comment:
Even if the password is correct, this could trigger. Probably should run this on wallet failure.

Reply:
fixed.